deploy/kubeturbo-operator/bundle/7.21.0/kubeturbo-operator.v7.21.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,210 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[{"apiVersion":"charts.helm.k8s.io/v1alpha1","kind":"Kubeturbo","metadata":{"name":"kubeturbo-release"},"spec":{"restAPIConfig":{"opsManagerPassword":"Turbo_password","opsManagerUserName":"Turbo_username"},"serverMeta":{"turboServer":"https://Turbo_server_URL","version":7.21}}}]'
```

Comment:
Nit: probably should include turbo credentials secret here too, now that you've introduced it?

Reply:
this is just an example and I am removing all the credentials, I think we should encourage users to use the secret and by default they can just create the default secretname